### DISCUSSION TITLE
A new collection maintainer

### DISCUSSION THREAD

**Discussion by Andersson007** [ansible-collections/ansible.mysql#547] [General] [answered]
Hey dear maintainers and the community!

I'd like to ask you if you folks are OK with inviting @betanummeric as a maintainers to this repository? (i.e. inviting with `commit`)
He's been actually in our crew already for a long time.
There's no need to mention all the great contributions he did to this collection and to community.postgresql, it was a lot!

(i'm not using the poll feature intentionally as it does not allow to see who voted)

@laurent-indermuehle @Jorge-Rodriguez @markuman @rsicart @bmalynovytch please put +1/+0/-1.

**Comment by Andersson007** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by rsicart** [comment]
+1

**Comment by Jorge-Rodriguez** [comment]
+1

**Comment by bmalynovytch** [comment]
+1

**Comment by laurent-indermuehle** [comment]
+1

**Comment by Andersson007** [comment, marked as answer]
@betanummeric invited you with `maintainer` (same as all other folks). I don't see that the invite is "pending" but please check your email box just to be on the safe side.
Congratulations and thanks again for your contributions!

Thanks everyone for voting! Locking the discussion.
